pkg/options/options.go

```diff
@@ -12,6 +12,7 @@ type ApnsOptions struct {
 	KeyId                 string `long:"apns-key-id" env:"APNS_KEY_ID" description:"Key ID associated with APNS credentials"`
 	TeamId                string `long:"apns-team-id" env:"APNS_TEAM_ID" description:"APNS Team ID"`
 	Topic                 string `long:"apns-topic" env:"APNS_TOPIC" description:"Topic to be used on all messages"`
+	Mode                  string `long:"apns-mode" env:"APNS_MODE" description:"Which APNS servers to deliver to, development or production"`
```

Comment:
You can add `choice:"development" choice:"production"` to the struct tag to restrict the available options

Reply:
I would also suggest adding a default. Otherwise this will fail to deploy, since nothing is configured right now.

Reply:
Good calls, added both.